mParticle-Singular/MPKitSingular.m

```diff
@@ -68,6 +74,11 @@ - (void)start{
         /*
          Start your SDK here. The configuration dictionary can be retrieved from self.configuration
          */
+        ddlLink = nil;
+        [Singular registerDeferredDeepLinkHandler:^(NSString *deeplink) {
+            ddlLink = [[NSDictionary alloc] initWithObjectsAndKeys:deeplink,SINGULAR_DEEPLINK_KEY, nil];
```

Comment:
@anismiles you should be invoking the deep link completion handler here, if not nil